### DISCUSSION TITLE
[Question][Hw1] About archive format

### DISCUSSION THREAD

**Discussion by mrpositron** [prosyslab-classroom/cs348-information-security#496] [Q&A]
Is it okay if I will use `.zip` format instead of `.tar`. On windows I don't have an option to archive to `.tar` file.

**Comment by KAIST-JongchanPark** [comment]
As far as I know, 7-zip in windows have `.tar` option

**Comment by mrpositron** [comment]
I have already submitted it as a `.zip` fie. Is it fine? Or should I resubmit it using grace days?

**Comment by jirheee** [comment]
Actually I want to ask the same question,,, I submitted as tar.gz format, and I mailed the TA but there is no response... Will there be penalties for not using .tar format...?
